crates/ruff_linter/src/rules/flake8_type_checking/helpers.rs

```diff
@@ -420,3 +434,24 @@ impl Transformer for QuoteAnnotator<'_> {
         }
     }
 }
+
+/// A [`Transformer`] class that rewrites all strings in an expression
```

Comment:
Ugh, this is on me -- it's Rust, not Python!

```suggestion
/// A [`Transformer`] struct that rewrites all strings in an expression
```

Reply:
Wow I didn't notice either!